src/environment/mod.rs

```diff
@@ -140,8 +140,8 @@ impl<N: Network> Environment for ClientTrial<N> {
         "159.223.124.150:4132", "137.184.192.155:4132", "147.182.213.228:4132", "137.184.202.162:4132", "159.223.118.35:4132",
         "161.35.106.91:4132", "157.245.133.62:4132", "143.198.166.150:4132",
     ];
-    const MINIMUM_NUMBER_OF_PEERS: usize = 11;
-    const MAXIMUM_NUMBER_OF_PEERS: usize = 31;
+    const MINIMUM_NUMBER_OF_PEERS: usize = 41;
+    const MAXIMUM_NUMBER_OF_PEERS: usize = 71;
```

Comment:
Can we increase these to just 25 and 50 if it must go up? My concern is performance overhead

Reply:
I saw no issues with these (and even larger) values when running a client; the client only handles the network, after all